### PR TITLE
Adds error message for list_user_permissions missing argument

### LINKED ISSUE #183: rabbitmqctl gives cryptic erlang error when list_user_permissions is missing arguments
Run `./scripts/rabbitmqctl list_user_permissions` without passing the `user` option

### PR DESCRIPTION
**Pull request by videlalvaro**
Fixes #183

### CODE REVIEW

File: src/rabbit_control_main.erl [hunk 1]
```diff
@@ -519,6 +519,9 @@ action(list_vhosts, Node, Args, _Opts, Inform, Timeout) ->
     display_info_list(call(Node, {rabbit_vhost, info_all, []}, Timeout),
                       ArgAtoms);
 
+action(list_user_permissions, _Node, _Args = [], _Opts, _Inform, _Timeout) ->
```
Comment (dumbbell) [inline src/rabbit_control_main.erl]: Perhaps `Args = [""]` could be caught here as well?

I'm thinking of scripts calling `rabbitmqctl list_user_permissions "$user"` but `$user` is not set.

Comment (videlalvaro) [inline src/rabbit_control_main.erl]: wouldn't the empty string be a problem with most commands actually? I think for that case we get an error like this:

```
Listing permissions for user [] ...
Error: no_such_user: 
```

which at least tells me something about the problem, not like `function_clause` before

Comment (michaelklishin) [inline src/rabbit_control_main.erl]: Yes, if we want to handle empty values, it needs to be a separate issue (not sure how many run into that).

Comment (dumbbell) [inline src/rabbit_control_main.erl]: Ok, the current error is fine. Disregard my comment.

**Pull request merged**